sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -417,7 +418,7 @@ MarginCalculationLauncher::computeLocalMargin(const boost::shared_ptr<LoadIncrea
         maxLoadVarForLoadIncrease = newVariation;
       LoadIncreaseResult liResultTmp;
       liResultTmp.resize(events.size());
-      findOrLaunchScenarios(baseJobsFile, events, toRunCopy, liResultTmp);
+      findOrLaunchScenarios(baseJobsFile, events, toRunCopy, results, liResultTmp);
```

Comment:
Why using results here?

Reply:
in the context of local margin, results = maximumVariationPassing